packages/commonwealth/client/scripts/views/modals/edit_topic_modal.tsx

```diff
@@ -51,9 +51,18 @@ export const EditTopicModal = ({
     featuredInSidebarProp,
   );
   const [name, setName] = useState<string>(nameProp);
+  const [image, setImage] = useState<string | null>(null);
 
   const { isAddedToHomeScreen } = useAppStatus();
 
+  const markdownImageRegex = useMemo(() => /!\[image\]\((.*?)\)/, []);
```

Comment:
why does this need to be memoized? It is just a constant I think (Just a regex)

Reply:
exactly, this should be outside of the component, preferably in some util file